db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_08_16_035105) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.integer "user_id"
+    t.integer "post_id"
+    t.string "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_comments_on_post_id"
+    t.index ["user_id"], name: "index_comments_on_user_id"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.integer "user_id"
+    t.integer "post_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_likes_on_post_id"
+    t.index ["user_id"], name: "index_likes_on_user_id"
+  end
```

Comment:
- [ ] Please change the user_id column to author_id for both `comment` and `likes` table just like you have done for the post table. The author_id column should point to the user table in your migration file. 

Please do not merge your code until it is approved.

Reply:
@bobb-Rob thank you for the feedback, in the diagram the only author_id column is for the posts table for the other tables it appears as user_id 
![image](https://github.com/PabloEGonz/blog-app/assets/68977631/b0890ec1-35d7-496a-88b0-5711dd592627)

And it is also specified in the requirements. It should only be for the posts table.
![image](https://github.com/PabloEGonz/blog-app/assets/68977631/a3d3cc4f-c8a6-4f71-a911-d94d3c984961)